dotenv-linter/src/cli/mod.rs

```diff
@@ -154,6 +154,8 @@ fn not_check_updates_flag() -> Arg {
     Arg::new("not-check-updates")
         .long("not-check-updates")
         .help("Doesn't check for updates")
+        .value_parser(clap::builder::BoolValueParser::new())
```

Comment:
I'd like to use this one: [BoolishValueParser](https://docs.rs/clap/latest/clap/builder/struct.BoolishValueParser.html)

Reply:
Updated here: https://github.com/dotenv-linter/dotenv-linter/pull/625/commits/005622de4a6c0a0df564c092a07baf4e3e395404#diff-345b89c66e65c7e3c0f07e0d58c74a65f44215fcdf6e24dfd32a11f89f7209e5R157